### PR TITLE
Implement MongoLog.

### PR DESCRIPTION
**Pull request by RageZBla**
Reference #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ root:
  - The Mongo class is deprecated and was not implemented in this library. If you
  are still using it please update your code to use the new classes.
 
+## MongoLog
+
+ - The [MongoLog](http://php.net/manual/en/class.mongolog.php) class does not log anything because the underlying driver does not offer a method to retrieve this
```
Comment (alcaeus) [inline README.md]: Could you truncate the lines to 80 characters? Thanks!

File: lib/Mongo/MongoLog.php [hunk 2]
```diff
@@ -144,5 +177,9 @@ public static function setModule($module) {}
      * @static
      * @return int Returns the modules currently being logged
      */
-    public static function getModule() {}
+    public static function getModule()
+    {
+        return self::$module;
+    }
+
```
Comment (alcaeus) [inline lib/Mongo/MongoLog.php]: Superfluous linebreak ;)

File: tests/Alcaeus/MongoDbAdapter/MongoLogTest.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        \MongoLog::setLevel(2);
+        $this->assertSame(2, \MongoLog::getLevel(2));
+    }
+
+    public function testModule()
+    {
+        \MongoLog::setModule(2);
+        $this->assertSame(2, \MongoLog::getModule(2));
+    }
+}
```
Comment (alcaeus) [inline tests/Alcaeus/MongoDbAdapter/MongoLogTest.php]: Missing newline ;)

**Comment by alcaeus** [comment]
On the way out, otherwise I'd fix the issues while merging. Looks good, will hopefully get to it tomorrow. Thanks!

File: README.md (hunk 1 above)
Comment (RageZBla) [inline README.md]: Yes, sorry

File: tests/Alcaeus/MongoDbAdapter/MongoLogTest.php (hunk 3 above)
Comment (RageZBla) [inline tests/Alcaeus/MongoDbAdapter/MongoLogTest.php]: Added, thanks!

**Pull request merged**